### PR TITLE
[KOGITO-3466] Avoid null values if none given as param and runtimes deploy skipped

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3466

### CODE REVIEW

File: Jenkinsfile.release [hunk 1]
```diff
@@ -131,15 +131,10 @@ pipeline {
                         deployProps = readDeployProperties(jobUrl)
                     }
 
-                    env.STAGING_REPOSITORY = params.ARTIFACTS_REPOSITORY != '' ? params.ARTIFACTS_REPOSITORY : deployProps['staging-repo.url']
-                    
-                    env.EXAMPLES_URI = params.EXAMPLES_URI != '' ? params.EXAMPLES_URI : deployProps['kogito-examples.pr.source.uri']
-                    env.EXAMPLES_REF = params.EXAMPLES_REF != '' ? params.EXAMPLES_REF : deployProps['kogito-examples.pr.source.ref']
+                    env.STAGING_REPOSITORY = params.ARTIFACTS_REPOSITORY ?: (deployProps['staging-repo.url'] ?: '')
+                    env.EXAMPLES_URI = params.EXAMPLES_URI != '' ?: (deployProps['kogito-examples.pr.source.uri'] ?: '')
+                    env.EXAMPLES_REF = params.EXAMPLES_REF != '' ?: (deployProps['kogito-examples.pr.source.ref'] ?: env.RELEASE_BRANCH)
```
Comment (Kevin-Mok) [inline Jenkinsfile.release]:
```suggestion
                    env.EXAMPLES_URI = params.EXAMPLES_URI ?: (deployProps['kogito-examples.pr.source.uri'] ?: '')
                    env.EXAMPLES_REF = params.EXAMPLES_REF ?: (deployProps['kogito-examples.pr.source.ref'] ?: env.RELEASE_BRANCH)
```

**Review by Kevin-Mok — CHANGES_REQUESTED** [review]

File: Jenkinsfile.release (hunk 1 above)
Comment (Ginxo) [reply Jenkinsfile.release]: I agree with @Kevin-Mok

**Review by Ginxo — APPROVED** [review]
Adding @Kevin-Mok proposal it LGTM

File: Jenkinsfile.release (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile.release]: nice catch :)

**Review by Kaitou786 — APPROVED** [review]

**Review by Kevin-Mok — APPROVED** [review]

**Pull request merged**